lms/djangoapps/courseware/entrance_exams.py

```diff
@@ -43,118 +43,16 @@ def user_can_skip_entrance_exam(user, course):
     return False
 
 
-def user_has_passed_entrance_exam(request, course):
+def user_has_passed_entrance_exam(user, course):
     """
     Checks to see if the user has attained a sufficient score to pass the exam
     Begin by short-circuiting if the course does not have an entrance exam
     """
     if not course_has_entrance_exam(course):
         return True
-    if not request.user.is_authenticated():
-        return False
-    entrance_exam_score = get_entrance_exam_score(request, course)
-    if entrance_exam_score >= course.entrance_exam_minimum_score_pct:
-        return True
-    return False
-
-
-# pylint: disable=invalid-name
-def user_must_complete_entrance_exam(request, user, course):
-    """
-    Some courses can be gated on an Entrance Exam, which is a specially-configured chapter module which
-    presents users with a problem set which they must complete.  This particular workflow determines
-    whether or not the user is allowed to clear the Entrance Exam gate and access the rest of the course.
-    """
-    # First, let's see if the user is allowed to skip
-    if user_can_skip_entrance_exam(user, course):
-        return False
-    # If they can't actually skip the exam, we'll need to see if they've already passed it
-    if user_has_passed_entrance_exam(request, course):
+    if not user.is_authenticated():
         return False
-    # Can't skip, haven't passed, must take the exam
-    return True
-
-
-def _calculate_entrance_exam_score(user, course_descriptor, exam_modules):
```

Comment:
**Note to reviewers:** The custom grading methods (`get_entrance_exam_score` and `_calculate_entrance_exam_score`) are now removed in favor of piggybacking on signals from the grades framework.